.claude/agents/openshift-cluster-expert.md

```diff
@@ -0,0 +1,49 @@
+---
+name: ocp-expert
```

Comment:
the file is "openshift-cluster-expert", then you use "ocp-expert". Can we use the same term everywhere and be consistent with the existent -sme pattern?

Maybe "ocp-admin-sme"?

Reply:
I think "ocp-troubleshooter-sme" is the one that captures better the use case / intent of the agent